lib/ui/pages/main/home/home_controller.dart

```diff
@@ -0,0 +1,66 @@
+import 'package:chat_flutter/model/group.dart';
+import 'package:chat_flutter/model/user.dart';
+import 'package:flutter/material.dart';
+
+class HomeController with ChangeNotifier{
+  HomeController();
+
+  Future<User> getMeById(String userId) async {
```

Comment:
基本Controllerのメソッドは戻り値を `Future<void>` にして、　Controllerのプロパティで Userを持ってあげます。
するとFuture型を処理せずともUserという型でWidgetに通知できるからです。
具体的には、
```suggestion 
  User user;
  Future<void> getMeById(String userId) async {
         await Future.delayed(Duration(seconds: 1));
         user = User(name: "test", imgUrl: "https://dot.asahi.com/S2000/upload/2019100100055_1.jpg");
         notifyListeners()
```